web/source/osk/oskBaseKey.ts

```diff
@@ -45,15 +47,15 @@ namespace com.keyman.osk {
           }
       }
 
+      let q = document.createElement('div');
+      q.className='kmw-key-label';
       if(x > 0) {
-        let q = document.createElement('div');
-        q.className='kmw-key-label';
         q.innerHTML=String.fromCharCode(x);
-        return q;
       } else {
         // Keyman-only virtual keys have no corresponding physical key.
-        return null;
+        // So, no text for the key-cap.
       }
+      return q;
     }
```

Comment:
Internal change:  base keys on the OSK now _always_ have a key-cap element.  It's just hidden by default and empty when no cap-text is available.